backend/docker_ops.py

```
@@ -1,383 +1,642 @@
-            logger.error(
-                f"No compose file found in {compose_dir} on {target_host}")
+    
+    async def create_network_on_target(self, network_info: NetworkInfo, target_host: str, 
+                                     ssh_user: str = "root", ssh_port: int = 22) -> bool:
+        """Create a Docker network on the target host using Docker API"""
+        try:
+            client = self.get_docker_client(target_host, ssh_user)
+            
+            # Check if network already exists
+            try:
+                existing_network = client.networks.get(network_info.name)
```

Ruff (0.11.9)

Local variable `existing_network` is assigned to but never used

Remove assignment to unused variable `existing_network`

(F841)